openupgrade_scripts/scripts/account_peppol/18.0.1.1/upgrade_analysis.txt

```diff
@@ -4,7 +4,7 @@ new model account_peppol.service.wizard [transient]
 new model peppol.registration [transient]
 ---Fields in module 'account_peppol'---
 account_peppol / account.move             / peppol_move_state (selection) : selection_keys removed: [canceled]
-account_peppol / res.company              / account_peppol_proxy_state (selection): selection_keys added: [receiver, sender, in_verification, smp_registration], removed: [canceled, sent_verification, pending, active, not_verified]
+account_peppol / res.company              / account_peppol_proxy_state (selection): selection_keys added: [smp_registration, receiver, in_verification, sender], removed: [not_verified, pending, sent_verification, active, canceled]
```

Comment:
It looks not ordered.

Reply:
Yeah, it seems random. Maybe `upgrade_analysis` should explicitly sort them (extra PR needed), and then accept the last update with this sorted.